automated/linux/ethernet/test-eth3.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+source "../test_format.sh"
+
+TEST_NAME="Test ETH-3"
```

Comment:
I suggest meaningful names for both the TEST_NAME and file name of test scripts and yaml files
e.g. test-eth-perf3, test-eth-ethtool, test-eth-ping, usw...